src/components/flag/flag.jsx

```diff
@@ -7,16 +7,20 @@ import CurrencyFlag from './flags/currencies';
 
 export const styles = theme => {
   const { palette } = theme;
-
+  const addBorder = props => !props.hideBorder && !props.round && !props.secondaryCountryCode;
   return {
+    container: {
+      lineHeight: 0,
```

Comment:
This is not needed. You could just add `vertical-align: bottom` to the SVG and skip this container div